GOVERNANCE.md

```diff
@@ -0,0 +1,114 @@
+- Overseeing the resolution and disclosure of security issues
+- Managing financial decisions related to the project
+
+In addition to the responsibilities listed above, this group is also responsible for bootstrapping a multi-stakeholder steering committee of 5-7 people to govern the project. The group is responsible for creating a steering committee Charter and shaping and executing the processes around selecting committee members. Once a steering committee is in place, the Spin Governance Committee will be dismantled and replaced with the Spin Steering Committee. The Spin Steering Committee should then establish additional governance structures as it sees fit (e.g., a Code of Conduct Committee for project moderation).
+
+The Spin Governance Committee have to match the following criteria:
+
+- Spin Governance Committee members MUST remain active on the project. If they are unresponsive for > 3 months, they will lose membership, unless a [supermajority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote) of the other governance members agrees to extend the period to be greater than 3 months
+
+The Spin Governance Committee will select a chair to set agendas, call meetings, and oversee the decision making process.
+
+### Decision Making
+
+The default decision making process is objection-free consensus except for changes to the Governance of the project which requires a supermajority agreement of the committee.
+
+Decision making scenarios should be promoted appropriately by the member overseeing the issue.
+
+In the extreme case that objection-free consensus cannot be reached after a reasonable amount of time and effort,
+a member can call for a supermajority vote from the committee.
+
+### Changes to Governance
+
+The process for changes to the GOVERNANCE.md file is as follows:
+
+1. Decide whether the change is substantial in nature. A substantial change is one that changes the project governance itself, instead of editorial changes providing clarifications, small bugfixes, etc
+2. Then,
+    - For substantial changes:
+        i. Open a new SIP for the changes
+        ii. Follow the [normal SIP process](https://github.com/fermyon/spin/blob/main/docs/content/sips/index.md), requiring a supermajority for acceptance
+    - For editorial changes:
+        i. Open a PR with the changes to GOVERNANCE.md
+        ii. Land the PR after getting at least two positive reviews, and waiting for at least 3 workdays across all time zones
+
+_Note: PRs landed without SIPs can be challenged as substantial for up to 2 weeks afterwards, at which point they have to be backed out, and the proposed changes have to go through the SIP process instead._
+
+1) Open an issue
+2) Open a pull request to this SIP describing changes and the GOVERNANCE.md file with proposed changes. 
+3) Spin Govenance Committee will review pull request. Pull request may be merged when a supermajority of Spin Governance Committee has approved the pull request.
```

Comment:
I think this needs to be removed, right?

Reply:
updated